### ISSUE TITLE
core.check_player_privs expects a player or playername as argument.

### ISSUE THREAD

**Issue by Git-Forked** [x2048/terraform#7]
**2021-03-12 12:36:29: ERROR[Main]: ServerError: AsyncErr: Lua: finishGenRuntime error from mod 'glooptest' in callback item_OnPlace(): ...ome/minetest/minetest/bin/../mods/terraform/init.lua:5: core.check_player_privs expects a player or playername as argument.
2021-03-12 12:36:29: ERROR[Main]: stack traceback:
2021-03-12 12:36:29: ERROR[Main]: 	[C]: in function 'error'
2021-03-12 12:36:29: ERROR[Main]: 	/home/minetest/minetest/bin/../builtin/game/misc.lua:11: in function 'check_player_privs'
2021-03-12 12:36:29: ERROR[Main]: 	...ome/minetest/minetest/bin/../mods/terraform/init.lua:5: in function 'privileged'
2021-03-12 12:36:29: ERROR[Main]: 	...ome/minetest/minetest/bin/../mods/terraform/init.lua:62: in function 'callback'
2021-03-12 12:36:29: ERROR[Main]: 	/home/minetest/minetest/bin/../builtin/game/item.lua:398: in function </home/minetest/minetest/bin/../builtin/game/item.lua:258>**

### **Minetest server info:** 
```
Minetest 5.4.0 (Linux)
Using Irrlicht 1.8.4
Using Lua 5.1.5
BUILD_TYPE=Release
RUN_IN_PLACE=0
USE_CURL=1
USE_GETTEXT=0
USE_SOUND=1
USE_FREETYPE=1
```
Error causes server to shutdown.

**Comment by x2048** [comment]
This happens when a non-player entity digs or places blocks when terraform mod is enabled.

**Comment by Git-Forked** [comment]
Having an excuse for a bug isn't quite the same as fixing it, or accounting for it in the code to safely ignore the dug node without a complete server crash.

**Comment by x2048** [comment]
Hey @Git-Forked,

Not sure what you mean by excuse. The bug was fixed in bb75b9a and the fix is available in v1.2. The comment above is merely a reproduction scenario for myself.

May I ask you to let me know if you still experience the issue?
